# DESAFIO 7
### Desenvolva um programa que leia as duas notas de um aluno, calcule e mostre a sua média.


In [10]:
notaA = float(input('Insira a 1° nota: '))
notaB = float(input('Insira a 2° nota: '))
média = (notaA + notaB)/2

print('A média entre {} e {} é igual a {:.3f}'.format(notaA, notaB, média))

Insira a 1° nota: 4.7
Insira a 2° nota: 8.3
A média entre 4.7 e 8.3 é igual a 6.500


###  {:.3f} 
Na parte do print, perceba que dentro das chaves tem esse código acima. Ele diz que terá apenas uma determinada quantidade de números após a vírgula. Neste caso 3 (6.500).
  O f significa que é número flutuante (float)